themes/SuiteP/include/MySugar/tpls/MySugar2.tpl

```diff
@@ -67,7 +67,7 @@
 <!-- Construct Dashlets -->
 <div id="pageContainer" class="yui-skin-sam">
     <div class="row" id="pageNum_{$activePage}_div">
-        <table width="100%">
+        <table width="100%" style="table-layout: fixed;">
```

Comment:
Do not use inline style attribute for css defs, please outsource into .scss file.